backend/auth.js

```diff
@@ -2,44 +2,55 @@
 // https://hapijs.com/tutorials/auth
 // https://hapijs.com/tutorials/plugins
 
-import { verify, b64ToStr } from '~/shared/functions.js'
-
+import { verifyShelterAuthorizationHeader } from '~/shared/domains/chelonia/utils.js'
 const Boom = require('@hapi/boom')
 
 exports.plugin = {
   name: 'gi-auth',
   register: function (server: Object, opts: Object) {
+    server.auth.scheme('gi-bearer', function (server, options) {
```

Comment:
Perhaps it is time to drop the `gi-` prefix and start using either `chel-` or `shelter-`?